static/js/public/snap-details/exapandable-distr-chart.js

```diff
@@ -0,0 +1,29 @@
+export default function initExpandableDistroChart() {
+  const moreEl = document.querySelector(".snapcraft-distro-chart__more");
+  const lessEl = document.querySelector(".snapcraft-distro-chart__less");
+  const detailsEl = document.querySelector(".snapcraft-distro-chart");
+
+  if (moreEl && lessEl && detailsEl && screen.width <= 772) {
```

Comment:
We could probably add the `u-hide` and `u-show` classes in the markup to hide/show the buttons on desktop and mobile and remove the `screen.width` conditional. That way if someone reduces their screen size and the buttons show, they will still work.
Currently, if you reduce the size of the page the list of distros just gets cut off and if you increase the size of the page, the buttons remain.